### PR TITLE
[Basic Action] (No.6) Unify basic action on VM & Storage Account & ResourceManagement

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/ResourceManagementNode.java [hunk 1]
```diff
@@ -99,30 +104,9 @@ public String getRgName() {
...
-        @Override
-        public int getPriority() {
-            return Sortable.LOW_PRIORITY;
-        }
-
-        @Override
-        public int getGroup() {
-            return Groupable.MAINTENANCE_GROUP;
-        }
+    @AzureOperation(value = "delete resource management", type = AzureOperation.Type.ACTION)
```
Comment (Flanker32) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/ResourceManagementNode.java:107]: extract value to `ActionConstants`?

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/deployments/DeploymentNode.java [hunk 2]
```diff
@@ -87,30 +87,14 @@ public String getSubscriptionId() {
...
 
-    // Show property action class
-    private class ShowDeploymentPropertyAction extends NodeActionListener {
-
-        @Override
-        protected void actionPerformed(NodeActionEvent e) throws AzureCmdException {
-            EventUtil.logEvent(EventType.info, ARM, SHOW_DEPLOYMENT_PROPERTY, null);
-            DefaultLoader.getUIHelper().openDeploymentPropertyView(DeploymentNode.this);
-        }
+    @AzureOperation(value = "show properties about Deployment of Resource Management", type = AzureOperation.Type.ACTION)
```
Comment (Flanker32) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/deployments/DeploymentNode.java:90]: extract value to `ActionConstants`?

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/ResourceManagementNode.java (hunk 1 above)
Comment (shenqianjin) [reply Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/ResourceManagementNode.java]: UPDATED

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/deployments/DeploymentNode.java (hunk 2 above)
Comment (shenqianjin) [reply Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/deployments/DeploymentNode.java:91]: updated.

**Review by Flanker32 — APPROVED** [review]

**Pull request merged**